modules/Users/tpls/EditViewFooter.tpl

```diff
@@ -78,39 +78,6 @@
                         </select>
                     </td>
                 </tr>
-                            {if !$HIDE_IF_CAN_USE_DEFAULT_OUTBOUND}
```

Comment:
$HIDE_IF_CAN_USE_DEFAULT_OUTBOUND template variable still there in:
/var/www/html/SuiteCRM1/modules/Emails/templates/emailOptions.tpl
/var/www/html/SuiteCRM1/modules/Users/UserEmailOptions.tpl
/var/www/html/SuiteCRM1/modules/Users/tpls/wizard.tpl
is that correct?